kernel/kernel-impl/src/main/java/org/sakaiproject/user/impl/PasswordPolicyProviderDefaultImpl.java

```diff
@@ -171,7 +169,7 @@ public PasswordRating validatePassword(String password, User user) {
         characterSets += isCharacterSetPresentInPassword(CHARS_LOWER, password);
         characterSets += isCharacterSetPresentInPassword(CHARS_UPPER, password);
         characterSets += isCharacterSetPresentInPassword(CHARS_DIGIT, password);
-        characterSets += isCharacterSetPresentInPassword(CHARS_SPECIAL, password);
```

Comment:
Is it worth keeping this old character set as well, so that punctuation classes and foreign characters are considered different.